src/main/java/net/minecraftforge/client/renderer/LevelRendererAdapter.java

```diff
@@ -0,0 +1,100 @@
+package net.minecraftforge.client.renderer;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.math.Matrix4f;
+import net.minecraft.client.Camera;
+import net.minecraft.client.renderer.LevelRenderer;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.util.profiling.ProfilerFiller;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.common.ForgeConfig;
+
+import java.util.Collection;
+
+/**
+ * Adapter class used to dispatch draw calls for vanilla and user-provided static
+ * render types during level rendering.
+ */
+public final class LevelRendererAdapter
```

Comment:
This isn't really an [adapter](https://en.wikipedia.org/wiki/Adapter_pattern) class, but more of a hooks class (in the same vein of `ForgeHooksClient`).